site/content/en/docs/contributing/rest-api-design.md

```diff
@@ -36,7 +36,7 @@ Common scheme for our REST API is `<VERB> [namespace] <objects> <id> <action>`.
 - Allow filtering, sorting, and pagination
 - Maintain good security practices
 - Cache data to improve performance
-- Versioning our APIs (e.g. `/api/v1`, `/api/v2`). It should be done when you
+- Versioning our APIs (e.g. `/api`, `/api/v2`). It should be done when you
```

Comment:
@Marishka17 , need to update the statement. It looks wrong now (e.g., `/api/v2`)